spec_parser/utils.py

```diff
@@ -174,10 +176,16 @@ def gen_rdf(self) -> None:
             for vocab_obj in vocabs.values():
                 vocab_obj._gen_rdf(g)
 
-        fname = path.join(self.args["out_dir"], f"model.ttl")
-        with safe_open(fname, "w") as f:
+        ttl_file_name = path.join(self.args["out_dir"], f"model.ttl")
```

Comment:
You don't really need f-string for the filename, do you?

Reply:
There are quite a few more of these already existing in the code, this can be fixed in a separate PR.

Reply:
fixed